android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/settings/SettingsFragment.java

```diff
@@ -84,7 +91,7 @@ private void deleteUserRequest() {
         };
 
         Response.ErrorListener onError = error -> {
-            if (error.networkResponse != null) {
+            if (null != error.networkResponse) {
                 switch (error.networkResponse.statusCode) {
                     case 401:
                         Log.e(TAG, "Unauthorized");
```

Comment:
adding a Log.e(TAG, "Unknown Error") entry to the default: branch would be useful for debugging purposses even though it is not part of this PR